docker-compose.keycloak.yml

```
@@ -0,0 +1,29 @@
+version: "3.5"
```

```suggestion
version: "3.2"
```
As the docker-compose version is 3.2 in other docker-compose files (required for CI), the version should be the same